code/datums/ai/generic_actions.dm

```diff
@@ -22,3 +22,43 @@
 /datum/ai_behavior/move_to_target/perform(delta_time, datum/ai_controller/controller)
 	. = ..()
 	finish_action(controller, TRUE)
+
+
+/datum/ai_behavior/break_spine
+	behavior_flags = AI_BEHAVIOR_REQUIRE_MOVEMENT
+	var/target_key
+	var/give_up_distance = 10
+
+
+/datum/ai_behavior/break_spine/perform(delta_time, datum/ai_controller/controller)
+	var/mob/living/batman = controller.blackboard[target_key]
+	var/mob/living/big_guy = controller.pawn //he was molded by the darkness
+
+	if(!batman.stat)
+		finish_action(controller, TRUE)
+
+	if(get_dist(batman, big_guy) >= give_up_distance)
+		finish_action(controller, FALSE)
+
+	if(big_guy.grab_state < GRAB_AGGRESSIVE) //not an aggro grab yet
+		big_guy.start_pulling(batman)
+		return
+
+
+	batman.visible_message("<span class='warning'>[batman] gets a slightly too tight hug from [big_guy]!</span>", "<span class='userdanger'>You feel your bones break as [big_guy] embraces you!</span>")
+
+	if(!iscarbon(batman))
+		batman.adjustBruteLoss(150)
```

Comment:
holy shit lmao

Reply:
![image](https://user-images.githubusercontent.com/20987591/110230095-d8e3a780-7f0e-11eb-87fe-84c880d798a8.png)

Reply:
Make them paraplegic after the hug.